tests/origocli/commands/events_test.py

```diff
@@ -24,3 +153,88 @@ def test_handler(mocker):
 
     assert read_input.called_once
     assert post_event.called_once_with(input_json, "dataset-id", "version-1")
+
+
+def test_resolve_dataset_uri():
```

Comment:
Jeg ville splittet denne i to tester, en for default-versjon og en for eksplisitt versjon, heller enn å optimlisere for gjenbruk av lista.